api/ruby/user-auditing/user_audit.rb

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env ruby
+
+# User Audit - Generated with Octokitchen https://github.com/kylemacey/octokitchen
+
+# Dependencies
+require "octokit"
+
+Octokit.configure do |kit|
+  kit.auto_paginate = true
+end
+
+client = Octokit::Client.new
+users = client.all_users
+n = 1
+puts "Importing users..."
+full_users = users.map { |u|
+  print "\r#{n}/#{users.count}"
+  n += 1
+  client.user(u.login) rescue nil;
+}
+
+suspended = full_users.select do |u|
+  next unless u
+  !u.suspended_at.nil? rescue false;
+end
+
+active = full_users.select do |u|
+  next unless u
+  u.suspended_at.nil? rescue false;
+end
+
+two_days = 172800
```

Comment:
I recommend using a self-documenting variable name (for increased clarity) and showing the calculation. Something like this:

```ruby
seconds_in_two_days = 60 * 60 * 48
```

Or this:

```ruby
seconds_in_two_days = 60 * # seconds in an minute
                      60 * # minutes in an hour
                      48   # hours in 2 days
```